tasks/page/processGerbers.js

```diff
@@ -152,6 +152,7 @@ if (require.main !== module) {
       }
       zipInfo.width = Math.max(stackup.top.width, stackup.bottom.width)
       zipInfo.height = Math.max(stackup.top.height, stackup.bottom.height)
+      zipInfo.layers = Math.min(stackupData.length, 10)
```

Comment:
It seems pcbStackup takes an array of layers. I am able to get the count, but limit it to 10 due to that being the limit for the PCBShopper URL param.

Reply:
I think this counts all layers rather than just copper ones. 

Regarding the limit, it would make more sense to limit it as you pass it to pcbshopper rather than here.

Reply:
In fact: I don't think we should limit it at all. Probably, if we ever get a 16 layer PCB Pcbshopper will have implemented support for more than 10 layers as well.

Reply:
Interesting, thought I was just adding a link, but now I am learning about PCB boards. :smile: